### PR TITLE
Added read and write TSV with headers. Closes #194

### LINKED ISSUE #194: read TSV with headers
I would like to be able to read TSV with headers, where each raw will become Map. So, I will be able to say raw["myheader"] instead of memorizing which number corresponds to which field in the workflow

### PR DESCRIPTION
**Pull request by vsmalladi**
### Checklist
- [x] Pull request details were added to CHANGELOG.md
- [x] Valid examples WDL's were added or updated to the SPEC.md (see the [guide](https://github.com/openwdl/wdl-tests/blob/main/docs/MarkdownTests.md) on writing markdown tests)

### CODE REVIEW

**Comment by wleepang** [comment]
I think this PR needs a rebase. There are a lot of changes in the diff unrelated to read/write of TSV that are coming from other PRs.

File: SPEC.md [hunk 1]
```diff
@@ -7951,18 +7951,26 @@ And `/local/fs/tmp/array.txt` would contain:
 ### `read_tsv`
 
 ```
-Array[Array[String]] read_tsv(File)
+Array[Array[String]] read_tsv(File), Array[Object] read_tsv(File, true), Array[Object] read_tsv(File, Boolean, Array[String])
```
Comment (jdidion) [inline SPEC.md]: Please put each signature on a different line, like how we've done it for the other overloaded functions

File: SPEC.md [hunk 2]
```diff
@@ -8010,14 +8070,31 @@ Example output:
 ### `write_tsv`
 
 ```
-File write_tsv(Array[Array[String]])
+File write_tsv(Array[Array[String]]|Array[Struct]), File write_tsv(Array[Array[String]], true, Array[String]), File write_tsv(Array[Struct], Boolean, Array[String])
```
Comment (jdidion) [inline SPEC.md]: Same here

File: SPEC.md [hunk 3]
```diff
@@ -8405,6 +8511,19 @@ CODE
 
 And `/local/fs/tmp/map.json` would contain:
 
+`first\nsecond\nthird`
+
+## File write_tsv(Array[Array[String]]|Array[Struct]), File write_tsv(Array[Array[String]], true, Array[String]), File write_tsv(Array[Struct], Boolean, Array[String])
```
Comment (jdidion) [inline SPEC.md]: Looks like there might have been a merge issue here

**Review by jdidion — CHANGES_REQUESTED** [review]

File: SPEC.md (hunk 1 above)
Comment (vsmalladi) [reply SPEC.md]: Fixed

File: SPEC.md (hunk 2 above)
Comment (vsmalladi) [reply SPEC.md]: Fixed

File: SPEC.md (hunk 3 above)
Comment (vsmalladi) [reply SPEC.md]: Fixed

**Review by jdidion — APPROVED** [review]
lgtm!

**Pull request merged**